.github/actions/spelling/allow.txt

```diff
@@ -450,6 +450,7 @@ VOS
 vso
 wapproj
 wchar
+wcout
```

Comment:
https://github.com/microsoft/winget-cli/pull/699#issuecomment-772702860

Reply:
This is not in the product code or tests directly, but in a standalone .exe used as a test installer. It was mixing narrow and wide chars and that was getting complicated, but it didn't seem worth it to add conversion between them there, so I moved it all to wide. It should be fine as it's independent of everything else.

Reply:
The only thing I'd note is that by using it here instead of, e.g. just telling the bot to ignore the file entirely, the bot will never warn about the next violation.

Think of it like page-fault protection.

Obviously reviewers are responsible for the content, so in theory some reviewer should catch future incursions.

(Just feedback from the peanut gallery.)